packages/gcf-utils/README.md

```diff
@@ -146,12 +146,13 @@ The following properties are logged:
 
 > Note: To have your Bot's custom metrics collected from Cloud Logging and processed with other metrics, please refer to the [data-processor documentation](../monitoring-system/data-processor#add-support-for-a-new-metric-from-an-existing-data-source)
 
-To log a metric with field `foo` and value `bar`:
+To log a metric for the event `foo`, with additional structured information
```

Comment:
I try to avoid the `foo` and `bar` terminology, because they make it wildly unclear what type of information you'd actually use.  Would `your-event-name` and `example-value`, or something similar work here?

Reply:
Stopped using `foo`.